indico/modules/users/schemas.py

```diff
@@ -39,6 +30,18 @@ def add_country_name(self, data, **kwargs):
         return data
 
 
+class UserSchema(mm.SQLAlchemyAutoSchema):
+    identifier = Function(lambda user: user.identifier)
```

Comment:
Is this even needed? I'd expect it to be autogenerated correctly... In any case, `identifier = fields.String()` should be enough if it isn't

Reply:
Not sure why it was there, seems to be generated correctly without..